cypress.base.config.ts

```diff
@@ -61,6 +61,7 @@ export const baseConfig: Cypress.ConfigOptions = {
         plugins: [
           new DefinePlugin({
             'process.env.AWX_API_PREFIX': JSON.stringify(env.AWX_API_PREFIX),
+            'process.env.AWX_GW_API_PREFIX': JSON.stringify(env.AWX_GW_API_PREFIX),
```

Comment:
here we are refereing to a GW API in un upstream project which we do not want to do